src/group_theory/quotient_group.lean

```diff
@@ -193,4 +193,44 @@ mul_equiv.of_bijective (ker_lift φ) ⟨ker_lift_injective φ, λ h, begin
   refl
 end⟩
 
+/-- If two normal subgroups `M` and `N` of `G` are the same, their quotient groups are
+isomorphic. -/
+@[to_additive "If two normal subgroups `M` and `N` of `G` are the same, their quotient groups are
+isomorphic."]
+def equiv_quotient_of_eq {M N : subgroup G} [M.normal] [N.normal] (h : M = N) :
+  quotient M ≃* quotient N :=
```

Comment:
They're not just isomorphic, they're equal `by subst h`

Reply:
Yes, but unless there is a lemma saying that `G = H → G ≃* H` I still need this isomorphism.

Reply:
You're right, constructing the mul_equiv means you also need a proof that the monoid structures are the same:
```
example {G H : Type*} [mh : monoid G] [mg : monoid H] (h : G = H) (hm : mh == mg): G ≃* H :=
by unfreezingI { subst h, have := eq_of_heq hm, subst this, }
```
What you have right now is probably easier to work with though